src/extension.ts

```diff
@@ -166,7 +166,13 @@ export function activate(context: vscode.ExtensionContext) {
     });
     initAsyncCommand(context, 'deploymentSlot.SwapSlots', async (node: DeploymentSlotNode) => {
         if (node) {
-            await node.swapDeploymentSlots(outputChannel);
+            try {
+                outputChannel.appendLine('Initializing deployment slot swap...');
```

Comment:
Is this necessary? When user executes the command, will there be a long wait without any UI indication of progress?

Reply:
After starting the swap, it can take about a minute for smaller apps to finish.  I haven't tested it with larger web apps, but I suspect it might take longer.